public/_partials/sections/Intro.jade

```diff
@@ -0,0 +1,25 @@
+#Intro.row
+  .col-xs-12
+    h1#Logo
+      img(src="assets/images/Logo.svg")
+      | PLTFRM
+    .intro.row
+      .col-xs-12.col-md-6.phones
+        img(src="assets/images/PLTRFM_TwoPhones.png")
```

Comment:
For `<img>`s, let's use the srcset attribute for retina support (http://brianflove.com/2014/08/07/retina-display-images/)